### ISSUE TITLE
MassBank compliance to record format

### ISSUE THREAD

**Issue by tsufz** [MassBank/MassBank-web#292]
Hi @sneumann and @meier-rene, I have still some issues with record format compliance. I tried to use the PROJECT tag for example. I didn't pass the validator. We should put some efforts in the compliance now. I can help their with testing cases etc.

Best,
Tobias

**Comment by tsufz** [comment]
@meier-rene and I agreed that we remove the `PROJECT` tag again and it will be added as an example for the comment tag.

**Comment by tsufz** [comment]
I am not so sure if the PROJECT tag is not getting more important in the future. [Project](https://schema.org/Project) is an upcoming feature in `Schemas` and thus it will be maybe attractive to the contributors to submit records related to projects in future. For example with respect to open data obligations in their projects.

**Comment by meier-rene** [comment]
Nice finding, but I'm unsure how a one line tag could help us in providing information for a complex Project data type with a lot of different fields.

**Comment by tsufz** [comment]
@meier-rene, you are right. We have the same issue with the `creator` entity (see #194). I suggest to map the full entities with restricted and useful items in the records for the machine readable part and to generate a one line tag from the entity. If the people don't provide detailed information, the one line tag will be passed. The latter solution is required anyway to ensure backwards compatibility with old  submissions. We cannot curate all records in one step even not with automation.

**Comment by tsufz** [comment]
I will work on the `creator` and `project` entities and provide a draft new record format version for further discussions (in a special branch on my fork).
